modules/ztp-image-based-upgrade-installing-oadp-with-gitops.adoc

```diff
@@ -0,0 +1,263 @@
+      namespace: openshift-adp
+      conditions:
+      - type: Succeeded
+        status: "True"
+        reason: InstallSucceeded
+----
+
+.Example directory structure
+[source,terminal]
+----
+├── kustomization.yaml
+├── sno
+│   ├── example-cnf.yaml
+│   ├── common-ranGen.yaml
+│   ├── group-du-sno-ranGen.yaml
+│   ├── group-du-sno-validator-ranGen.yaml
+│   └── ns.yaml
+├── source-crs
+│   ├── custom-crs
+│   │   ├── OadpSubscriptionNS.yaml
+│   │   ├── OadpSubscriptionOperGroup.yaml
+│   │   ├── OadpSubscription.yaml
+│   │   ├── OadpOperatorStatus.yaml
+----
+--
+
+. Add the CRs to your common `PolicyGenTemplate`:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  name: "example-common-latest"
+  namespace: "ztp-common"
+spec:
+  bindingRules:
+    common: "true"
+    du-profile: "latest"
+  sourceFiles:
+    - fileName: custom-crs/OadpSubscriptionNS.yaml
+      policyName: "subscriptions-policy"
+    - fileName: custom-crs/OadpSubscriptionOperGroup.yaml
+      policyName: "subscriptions-policy"
+    - fileName: custom-crs/OadpSubscription.yaml
+      policyName: "subscriptions-policy"
+    - fileName: custom-crs/OadpOperatorStatus.yaml
+      policyName: "subscriptions-policy"
+[...]
+----
+
+. Create the `DataProtectionApplication` CR and the S3 secret.
+
+.. Create the following CRs in your `source-crs/custom-crs` directory:
++
+--
+.DataProtectionApplication.yaml
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1
+kind: DataProtectionApplication
+metadata:
+  name: dataprotectionapplication
+  namespace: openshift-adp
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "100"
+spec:
+  configuration:
+    restic:
+      enable: false <1>
+    velero:
+      defaultPlugins:
+        - aws
+        - openshift
+      resourceTimeout: 10m
+  backupLocations:
+    - velero:
+        config:
+          profile: "default"
+          region: minio
+          s3Url: $url
+          insecureSkipTLSVerify: "true"
+          s3ForcePathStyle: "true"
+        provider: aws
+        default: true
+        credential:
+          key: cloud
+          name: cloud-credentials
+        objectStorage:
+          bucket: $bucketName <2>
+          prefix: $prefixName <2>
+status:
+  conditions:
+  - reason: Complete
+    status: "True"
+    type: Reconciled
+----
+<1> The `spec.configuration.restic.enable` field must be set to `false` for an image-based upgrade because persistent volume contents are retained and reused after the upgrade.
+<2> The bucket defines the bucket name that is created in S3 backend. The prefix defines the name of the subdirectory that will be automatically created in the bucket. The combination of bucket and prefix must be unique for each target cluster to avoid interference between them. To ensure a unique storage directory for each target cluster, you can use the {rh-rhacm} hub template function, for example, prefix: {{hub .ManagedClusterName hub}}.
```

Comment:
```suggestion
<2> The bucket defines the bucket name that is created in S3 backend. The prefix defines the name of the subdirectory that will be automatically created in the bucket. The combination of bucket and prefix must be unique for each target cluster to avoid interference between them. To ensure a unique storage directory for each target cluster, you can use the {rh-rhacm} hub template function, for example, prefix: `{{hub .ManagedClusterName hub}}`.
```